### ISSUE TITLE
Panels sometimes spawn at strange locations.

### ISSUE THREAD

**Issue by karunakaruna** [msub2/silk-brush#19]
Several panels are created not relative to the viewer. 
The 'Advanced Options'/More Options panel spawns facing downwards. 

I believe if the panel is grabbed and moved it cannot be interacted with, so the user must adjust their head to meet see the  panel and interact.

This also happened with the 'capture screenshot for save' window.

**Comment by msub2** [comment]
Yeah, this is probably one of the next big issues I'm gonna tackle. I have a couple theories on why it may be doing this (I sort of bypassed a bit of the VrSystem setup that was being used originally), but I'll need to dig into it.
